src/components/SecretsDeleteModal/SecretsDeleteModal.scss

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '../../scss/vars';
+
+.deleteModal {
+
+  align-items: baseline !important;
```

Comment:
avoid `!important` as it makes styles very difficult to override, instead favour a more specific selector
also try to use the design tokens where possible, for example referencing colours https://www.carbondesignsystem.com/guidelines/color/usage

instead of adding custom styles per modal we should try to keep things generic to ensure consistency across the experience, and use the Carbon styles where possible. If we do need to override the `.bx-*` styles in limited cases, we can do that, but we should try to work with the Carbon styles first